test/test-v8debugapi.ts

```diff
@@ -589,9 +611,11 @@ describe('v8debugapi', function() {
           assert(((localsVal as any).status as any).description.format.match(
             'Locals and arguments are only displayed.*config.capture.maxExpandFrames=0'
             ));
-          api.clear(bp);
           config.capture.maxExpandFrames = oldCount;
```

Comment:
This line might need to be inside the callback to `api.clear`.  That is, the change to `config.capture.maxExpandFrames` could possibly happen before the call to `api.clear` runs in which case it is using the wrong value of `config.capture.maxExpandFrames`.